toml.abnf

```diff
@@ -212,14 +212,13 @@ std-table-close = ws %x5D     ; ] Right square bracket
 
 ;; Inline Table
 
-inline-table = inline-table-open inline-table-keyvals inline-table-close
+inline-table = inline-table-open [ inline-table-keyvals ] inline-table-close
```

Comment:
This change means that at least one key/value pair is required - the old way of expressing it allowed for "empty" tables.

Reply:
@bitwalker New way allows tables that do not have `inline-table-keyvals` at all. Is it not the same as "empty tables"?

Reply:
Ah, yep, you're right, I was looking at the old definitions of `inline-table-*` and forgot about the `[]` around `inline-table-keyvals` up here.